src/isPrime.py

```diff
@@ -0,0 +1,18 @@
+import sys
+
+num = sys.argv[1]
+result = 1
+
+
+def isPrime(number):
+    for i in range(2, number):
+        if number % i == 0:
+            global result
```

Comment:
I know a bit earlier they show you how to use global and nonlocal to change the scoping, but you honestly should avoid doing it. It's not considered good practice and it's mostly there as an escape hatch for edge cases. Usually you should allow scope to work as intended.